portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -93,6 +96,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
+    Document doc =
+        Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+
+    System.out.println("This is the sentiment score" + score);
```

Comment:
For web servlets, use logging instead of System.out.println (I don't remember how to do GAE logging off the top of my head, but you can try googling for it).

But maybe remove the print ln since it won't help with debugging a live release of the server -- and datastore should have the sentiment score for every comment, right?